### ISSUE TITLE
Release number is int, not string

### ISSUE THREAD

**Issue by peterjaap** [justbetter/magento2-sentry#46]
Version 2.0.0

```
Exception #0 (Symfony\Component\OptionsResolver\Exception\InvalidOptionsException): The option "release" with value 1576503498 is expected to be of type "null" or "string", but is of type "integer".
```
```
cat pub/static/deployed_version.txt 
1576503498%                                                    
```

Should be cast to string before returning it to sentry SDK I guess..

**Comment by rakibabu** [comment]
Hi @peterjaap ,

Thanks for reporting, I'm not using the deployed_version so feel free to fix this in a PR.

**Comment by ctadlock** [comment]
This happens when the extension is first deployed.

**Comment by indykoning** [comment]
This should be fixed in the new version 2.2.3 it will be up to download soon.
